webapp/src/components/App/Singlestudy/NavHeader/NavHeaderCommands.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Tooltip, Typography, Chip, Button } from "@mui/material";
+import HistoryOutlinedIcon from "@mui/icons-material/HistoryOutlined";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import { useTranslation } from "react-i18next";
+import { StudyMetadata, StudyType } from "../../../../common/types";
+import { toggleFavorite } from "../../../../redux/ducks/studies";
+import StarToggle from "../../../common/StarToggle";
+import useAppDispatch from "../../../../redux/hooks/useAppDispatch";
+import useAppSelector from "../../../../redux/hooks/useAppSelector";
+import { isCurrentStudyFavorite } from "../../../../redux/selectors";
+
+interface Props {
+  study: StudyMetadata | undefined;
+  isExplorer: boolean | undefined;
+  onCopyId: () => Promise<void>;
+  onUnarchive: () => Promise<void>;
+  onLaunch: VoidFunction;
+  onOpenCommands: VoidFunction;
+}
+
+function NavHeaderCommands({
```

Comment:
Not a probleme because the component is inside the folder NavHeader. Long names can be avoided here. 
All component folders have an index.tsx for example, and there is no confusion. When we search a file with VS Code, the folder name is include.